tools/server_settings_replicator/server_settings_replicator.rb

```diff
@@ -1,11 +1,19 @@
 class ServerSettingsReplicator
   def self.replicate(server, path_string, dry_run = false)
+    disallowed_path = ['server', 'server/host', 'server/hostname']
+    raise "Not allowed to replicate path #{path_string}" if disallowed_path.include?(path_string)
```

Comment:
What happens if we have want to set "ntp/server"?  does the "server" value block us setting that?
Does this also block all settings in the server section?  For example, "server/timezone"?

Reply:
Actually, I'm wondering if it also blocks "session/memcache_server"... not sure you'd ever change it but still, it looks like we have to be more specific about what settings paths we want to block.

Reply:
It only blocks exact matches. Perhaps is would be beter to remove the setting from replication that would mess up your cluster